clerk/sessions.go

```diff
@@ -113,3 +118,15 @@ func (s *SessionsService) Verify(sessionId, token string) (*Session, error) {
 	}
 	return &sessionResponse, nil
 }
+
+func (s *SessionsService) CreateTokenFromTemplate(sessionId, templateSlug string) (*SessionToken, error) {
+	sessionUrl := fmt.Sprintf("%s/%s/token/%s", SessionsUrl, sessionId, templateSlug)
```

Comment:
Same here for renaming to  `sessionURL` and `sessionID`. :wink:

https://go.dev/talks/2014/names.slide#5